### PR TITLE
Bugfix/do not try to export suomifi theme function

### LINKED ISSUE #194: Exported suomifiTheme is now a function and it was an object
## Expected Behavior or Possible Solution

Export object

## Actual Behavior and Screenshots

https://github.com/vrk-kpa/suomifi-ui-components/blob/develop/src/core/theme/index.ts#L49

## Steps to Reproduce the Problem

`import { suomifiTheme } from 'suomifi-ui-components';`

### PR DESCRIPTION
**Pull request by J-Kallunki**
Do not try to export suomifiTheme function outside the library as it uses singleton to get theme from tokens. Exported defaultTokens contains also internal tokens so everything is still available.

Fix so that suomifiTheme function understands also given customized internalTokens.

Closes #194

### CODE REVIEW

File: src/core/theme/index.ts [hunk 1]
```diff
@@ -47,11 +48,13 @@ export const defaultTokens = {
...
-  spacing,
-  typography,
-}: SuomifiTokens = defaultTokens) => ({
+  colors = defaultTokens.colors,
+  spacing = defaultTokens.spacing,
+  typography = defaultTokens.typography,
+  ...internalTokensProp
+}: Partial<SuomifiTokens & DefaultInternalTokens> = defaultTokens) => ({
   ...internalTokens,
+  ...internalTokensProp,
```
Comment (aappoalander) [inline src/core/theme/index.ts]: What is this and where is this used / where does this come from?

Comment (J-Kallunki) [reply src/core/theme/index.ts]: This is the internal tokens that are customized. You can highlight it with IDE and typescript will tell what it contains. I'll update JSDOC for the function

Comment (LJKaski) [reply src/core/theme/index.ts]: Great. That's what I figured as well, but updating the JSDOC will definitely make it easier to see.

Comment (aappoalander) [reply src/core/theme/index.ts]: A bit difficult to follow what instances / types get returned, maybe rename internalTokensProp to be more descriptive when considering the use. In addition, documenting the return values and types might help.

**Pull request merged**